src/Cool/DidYouKnow/didYouKnowModal.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include "cereal/types/chrono.hpp"
+class DidYouKnowModal {
+public:
+    DidYouKnowModal() = default;
+
+    void open();
+    void prepare_next_tip();
+
+    bool is_open() const;
+
+    void display_modal_content();
+
+    void all_tips();
```

Comment:
This should not be in the public API, it should only be called internally by the main imgui window of this class